src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -151,6 +151,8 @@ const ProjectItemWrapper = styled.section`
       @media screen and (max-width: 550px) {
           img {
             padding: 1.65rem 0;
+			padding-left: 0.8rem;
+			padding-right: 0.5rem;
```

Comment:
It would be better to aggregate this in one line `padding: 1.65rem 0.5rem 1.65rem 0.8rem;`

Reply:
@Saksham294 ⬆